libopendmarc/opendmarc_policy.c

```diff
@@ -1718,6 +1734,11 @@ opendmarc_policy_to_buf(DMARC_POLICY_T *pctx, char *buf, size_t buflen)
 		if (strlcat(buf, pctx->dkim_domain, buflen) >= buflen) return E2BIG;
 	if (strlcat(buf, "\n", buflen) >= buflen) return E2BIG;
 
+	if (strlcat(buf, "DKIM_SELECTOR=", buflen) >= buflen) return E2BIG;
+	if (pctx->dkim_selector != NULL)
```

Comment:
I would combine this line and the next into a single "if".

Reply:
That should be apart of a general cleanup. You can't see it in the diff but there are many more existing examples of this format above and below this point.